### ISSUE TITLE
Ruby 2.7.0.dev doesn't play nicely with rubocop

### ISSUE THREAD

**Issue by luke-hill** [rubocop-hq/rubocop#6621]
Copy of https://github.com/rubocop-hq/rubocop/issues/6412, but for the latest ruby

## Expected behavior
Rubocop Runs.

## Actual behavior
Get an error during Yaml Loading with an invalid keyword `whitelisted_classes`
```
unknown keywords: whitelist_classes, whitelist_symbols
/home/travis/.rvm/rubies/ruby-head/lib/ruby/2.7.0/psych.rb:328:in `safe_load'
/home/travis/.rvm/gems/ruby-head/gems/rubocop-0.60.0/lib/rubocop/config_loader.rb:185:in `yaml_safe_load'
```

## Steps to reproduce the problem
https://travis-ci.org/natritmeyer/site_prism/jobs/474365429 - This is running against ruby-head on travis

## RuboCop version
0.60.0

**Comment by bbatsov** [comment]
Try 0.62.

**Comment by luke-hill** [comment]
I'll give that a whirl in the next couple of days and update.

**Comment by koic** [comment]
This issue resolved by #6470. RuboCop 0.62 (and 0.61) including this has been released.
